demos/espressif/esp32_devkitc_esp_wrover_kit/make/sdkconfig

```diff
@@ -396,6 +635,8 @@ CONFIG_OPENSSL_ASSERT_EXIT=
 #
 CONFIG_ESP32_PTHREAD_TASK_PRIO_DEFAULT=5
 CONFIG_ESP32_PTHREAD_TASK_STACK_SIZE_DEFAULT=3072
+CONFIG_PTHREAD_STACK_MIN=768
+CONFIG_PTHREAD_MUTEX_TIMEDLOCK=
 
 #
 # SPI Flash driver
```

Comment:
Do we still want these settings set to y?  Is SPIFFS being used still by anything?   Is there a downside to having these settings enabled?

Reply:
These are just additional option that existed before. If it needs to be changed, it would be in a separate PR.

Reply:
There is no downside, but yes can definitely be removed (since there is no default user)